drivers/gpio/gpio_mchp_xec_v2.c

```diff
@@ -81,103 +82,118 @@ static inline void xec_mask_write32(uintptr_t addr, uint32_t mask, uint32_t val)
 	sys_write32(r, addr);
 }
 
-/*
- * notes: The GPIO parallel output bits are read-only until the
- * Alternate-Output-Disable (AOD) bit is set in the pin's control
- * register. To preload a parallel output value to prevent certain
- * classes of glitching for output pins we must:
- * Set GPIO control AOD=1 with the pin direction set to input.
- * Program the new pin value in the respective GPIO parallel output
- * register.
- * Program other GPIO control bits except direction.
- * Last step set the GPIO control register direction bit to output.
+/* NOTE: gpio_flags_t b[0:15] are defined in the dt-binding gpio header.
```

Comment:
Conventionally, block comments in Zephyr usually keep `/*` on a separate line.

Also, please use `b[15:0]` to be consistent.

Reply:
Followed the block comments coding-style.